src/app/cluster/details/cluster/template.html

```diff
@@ -68,7 +68,8 @@
       <span>Open Dashboard</span>
     </a>
 
-    <button *ngIf="(settingsService.adminSettings | async)?.enableOIDCKubeconfig"
+    <button *ngIf="(settingsService.adminSettings | async)?.enableOIDCKubeconfig &&
```

Comment:
I think its better if we move subscription to `ts` file and create a class property for `adminSettings`.